doc/design.md

```diff
@@ -201,7 +207,7 @@ Returns: Search results
     * tbd
 
 
-**`GET`** `/stats`
+**`GET`** `/api/<version>/stats`
```

Comment:
Hey Arun! @a-ilango please let me know if this comment isn't helpful, definitely not trying to tell you what to do, but I thought I'd just share my thoughts. I'm taking a look at your PR ahead of tonight's Meetup to get a sense of where the project is and how auth is going to fit in with the frontend code.

I'd recommend not versioning the API unless we're planning to support multiple and/or 3rd party clients (I don't think we are). Versioning is great in those circumstances since you can make changes without breaking things. But since, AFAIK, we just have the one React JS client, I think it'll be better and simpler to not version the API and instead make changes to the client and the server at the same time. It will save time and be less complex. Just my $0.02.

Reply:
@maxkwallace, @erikjasso and I had an offline discussion regarding this and we decided to keep the versioning. In case there's a plan to create a public API in the future, the clients would be ready to handle the version and it should probably not be hard to handle it now.